honor --sysconfdir during runtime

mrbaseman opened the pull request:
as already mentioned in issue https://github.com/adrienverge/openfortivpn/issues/75 when running openfortivpn without any arguments it always searched the config file at the hard coded location /etc/openfortivpn/config
This pull request inserts the values for prefix and sysconfdir from configure time into the code via preprocessor defines. 
Previously, I have merged back the update to the man page etc. into my master branch, hoping that this doesn't disturb merging in the new changes. If it is better to delete this branch and fork again for the next pull-request, just drop me a message.

adrienverge commented on doc/openfortivpn.1.in:
```diff
@@ -126,17 +126,17 @@ username = foo
+user-cert = @SYSCONFDIR@/openfortivpn/user-cert.pem
 .br
-user-key =/etc/openfortivpn/user-key.pem
+user-key =@SYSCONFDIR@/openfortivpn/user-key.pem
```
Could you add a space after `=` while you're at it?

mrbaseman replied:
done

adrienverge commented on src/main.c:
```diff
@@ -100,7 +101,7 @@ int main(int argc, char **argv)
 	int ret = EXIT_FAILURE;
 	struct vpn_config cfg;
-	char *config_file = "/etc/openfortivpn/config";
+	char *config_file = ""SYSCONFDIR"/openfortivpn/config";
```
useless `""` at start, please remove

adrienverge commented on configure.ac:
```diff
@@ -36,5 +36,10 @@ AC_FUNC_MALLOC
 AC_FUNC_REALLOC
 AC_CHECK_FUNCS([bzero gethostbyname inet_ntoa memmove memset memmem select socket strerror strstr strtol])
 
+AC_DEFINE_UNQUOTED([PREFIX],["$prefix"])
```
Why this? Is it used somewhere?

mrbaseman replied:
this was left over from previous attempts to address the issue, removed now.

adrienverge commented on configure.ac:
```diff
@@ -36,5 +36,10 @@ AC_FUNC_MALLOC
 AC_CHECK_FUNCS([bzero gethostbyname inet_ntoa memmove memset memmem select socket strerror strstr strtol])
 
+AC_DEFINE_UNQUOTED([PREFIX],["$prefix"])
+AC_DEFINE_UNQUOTED([SYSCONFDIR],["`echo $(eval echo $sysconfdir)`"])
```
Isn't `echo $sysconfdir` enough? (instead of `echo $(eval echo $sysconfdir)`)

mrbaseman replied:
for `configure --sysconfdir` it would be enough, but for `configure --prefix` a simple `echo $sysconfdir` would give `${prefix}/etc` for `@SYSCONFDIR@`. The `eval` is needed here, but I could simplify this to `` `eval echo $sysconfdir` ``. I have chosen the backtick notation for better portability, because we need just one level of execution now (even though the `$( ... )` notation should be widely supported nowadays).

adrienverge commented:
Great, thanks!

adrienverge commented:
Last thing, could you rebase and squash to have one clean commit to merge?

mrbaseman commented:
I have squashed and rebased it now, and in addition:
- updated the date in the footer of the man page
- removed a trailing blank character from two lines in the man page

adrienverge commented:
Great, thanks!
